server/config.js

```diff
@@ -1,7 +1,15 @@
 const args = require("args-parser")(process.argv);
 const demoMode = args["demo"] || false;
 
+const badgeConstants = {
+    naColor: "#999",
+    defaultUpColor: "#66c20a",
+    defaultDownColor: "#c2290a",
+    defaultPingColor: "blue",  // as defined by badge-maker / shields.io
+};
+
 module.exports = {
     args,
-    demoMode
+    demoMode,
+    badgeConstants
```

Comment:
```suggestion
    badgeConstants,
```

Reply:
Is a dangling comma sth you would like to see everywhere ( multiline )? 
I was trying to stick to the current code style  ...
I will adjust my code :)